### PR TITLE
Allow filtering daemon sets by pod

### PR DESCRIPTION
**Pull request by drcapulet**

### CODE REVIEW

**Comment by rudle** [comment]
I've been doing `| grep` but this works too

File: bin/p2-dsctl/main.go [hunk 1]
```diff
@@ -56,6 +56,7 @@ var (
 	getID  = cmdGet.Arg("id", "The uuid for the daemon set").Required().String()
 
 	cmdList = kingpin.Command(CmdList, "List daemon sets.")
+	listPod = cmdList.Flag("pod", "The pod ID on the daemon set").String()
```
Comment (rudle) [inline bin/p2-dsctl/main.go]: typo: "of the daemon set"

**Comment by drcapulet** [comment]
I've been using `grep` as well but doesn't work as well when you've got pods that share a piece of the name.

**Comment by mpuncel** [comment]
fine for now, labels might be a more elegant solution later on but easy to change

**Pull request merged**